.vscode/settings.json

```diff
@@ -2,5 +2,14 @@
   "python.testing.pytestArgs": ["tests", "-vv"],
   "python.testing.unittestEnabled": false,
   "python.testing.pytestEnabled": true,
-  "pylint.args": ["--rcfile=pylintrc"]
+  "pylint.args": ["--rcfile=pylintrc"],
+  "python.analysis.autoImportCompletions": true,
+  "[python]": {
```

Comment:
This one is nice as well 😎 👍 
```py
    "editor.rulers": [88, 120],
```
or just 120.

Reply:
Yeah, I have it in many projects, but since I'm just monkey-patching an automatically generated thing, didn't go too much in the details 😄